### PR TITLE
profiles dir env var or default

### PR DESCRIPTION
**Pull request by JasonGluck**
Fix Issue  #4841.

### CODE REVIEW

**Comment by greg-finley** [comment]
@JasonGluck Thanks so much! Are you confident in this change? (Just wondering why marked as draft)

File: plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py [hunk 1]
```diff
@@ -193,7 +194,7 @@ def _get_profiles_dir(self):
                 self.sqlfluff_config.get_section(
                     (self.templater_selector, self.name, "profiles_dir")
                 )
-                or flags.PROFILES_DIR
+                or (os.getenv("DBT_PROFILES_DIR") or default_profiles_dir())
```
Comment (greg-finley) [inline plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py:197]: outer parens maybe unnecessary but also fine

**Comment by JasonGluck** [comment]
> @JasonGluck Thanks so much! Are you confident in this change? (Just wondering why marked as draft)

Tested this locally  both the with envvar set and without (default) and it worked. Put in draft because I have not yet contributed and wanted to make sure I was following the process properly.

**Comment by elyobo** [comment]
Thanks @JasonGluck!

**Pull request merged**